### PR TITLE
Eric update UI

### PR DESCRIPTION
**Pull request by IamEricYou**
Now we can choose the deck (Regular or Spanish) when the page is loaded.

and 'Help' tab is updated.

### CODE REVIEW

**Comment by IamEricYou** [comment]
Please let me know if you find any kind of bugs from this.

(+ I marked the Aces.jpg as a comment which causes the error)

File: src/main/java/assets/css/AcesUp.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 body {
   font-family: "Helvetica Neue", Helvetica, sans-serif;
-  background-image: url('Desktop/wallpapers/Aces.jpg');
```
Comment (vasquezt) [inline src/main/java/assets/css/AcesUp.css]: Thanks for taking out the file, maybe we could go about including it as a local file instead and referencing it from the directory?

File: src/main/java/assets/css/AcesUp.css [hunk 2]
```diff
@@ -94,15 +94,25 @@ button {
...
 }
 
-#deckBox {
-    background-color: red;
-    margin: 20px;
-    width: 350px;
-    border: 3px solid black;
-    left: 25%;
-
+.modal_deckBox{
```
Comment (vasquezt) [inline src/main/java/assets/css/AcesUp.css]: I like the way you changed the name to fit in the styling of our other code

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 3]
```diff
@@ -142,9 +141,16 @@ <h3 id="deckBox">
                <br><b>5.</b>When there are no more cards to move or remove, deal out the next four cards from the deck face-up onto each pile.
                <br><b>6.</b> Repeat Step 2, using only the visible, or top, cards on each of the four piles.
                <br><b>7.</b> When the last four cards have been dealt out and any moves made, the game is over. The fewer cards left in the tableau, the better. To win is to have only the four aces left.
+               <br><br><br>
+               <b>Rule (Spanish Deck)</b>
+               <br><b>1.</b> Spanish deck uses the following suits: bastos (clubs), oros (golden coins), copas (cups), and espadas (swords).
```
Comment (vasquezt) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: While this works as a way to implement the writing, potentially we could look into making this more concise in the code, or reference an outside file for the text

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 4]
```diff
@@ -195,6 +217,15 @@ <h3 id="deckBox">
             modal_credit.style.display = "none";
         }
     }
+    
+    window.onclick = function(event) {
```
Comment (vasquezt) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: I like the way you implemented the deck selection as a event trigger as opposed to the awkward button setup i had before

**Pull request merged**